pkg/instance/instance.go

```diff
@@ -58,14 +58,8 @@ func (i *Info) UpgradeRequired() bool {
 		return false
 	}
 
-	// Version annotation not being present means that the node has been created but not fully configured.
-	// The upgrade process is not required, the node should just be configured normally.
-	_, present := i.Node.GetAnnotations()[metadata.VersionAnnotation]
-	if !present {
-		return false
-	}
-
-	// Version annotation has an incorrect value, this was configured by an older version of WMCO and should be
+	// Version annotation is not present or has an incorrect value, means this instance
```

Comment:
How does this commit relate to adding Server 2022 support?

Reply:
As per our discussion, we are going to focus on the issue with the EC2 metadata endpoint not being available. Removed this commit as it was proposed as a walk-around.